### PR TITLE
#169534480  Built Signup endpoint

### PR DESCRIPTION
**Pull request by T2Wil**
#### What does this PR do?
- Modify db connection in controllers
#### Description of Task to be completed?
- create db connection
- write signup controller  instructions
- handle signup endpoint errors
- remove unnecessary db methods
- signup endpoint tests
#### How should this be manually tested?
* have Postman
* clone the repository
* cd into directory
* run ``npm run start:dev``

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#169534480 
#### Screenshots (if appropriate)
image

image

### CODE REVIEW

File: server/controllers/authController.js [hunk 1]
```diff
@@ -1,93 +1,61 @@
+import dotenv from 'dotenv';
+import { Pool } from 'pg';
```
Comment (mcaleb808) [inline server/controllers/authController.js]: @T2Wil  this should be in the database file

File: server/controllers/authController.js [hunk 2]
```diff
@@ -1,93 +1,61 @@
...
+      firstName, lastName, email, password,
+    } = req.body;
+    const createUsersTableReq = `CREATE TABLE IF NOT EXISTS 
+    users(
+      id UUID PRIMARY KEY,
+      firstName VARCHAR(128) NOT NULL,
+      lastName VARCHAR(128) NOT NULL,
+      email VARCHAR(128) UNIQUE NOT NULL,
+      password VARCHAR(128) NOT NULL
+      )`;
```
Comment (mcaleb808) [inline server/controllers/authController.js]: @T2Wil  this should be in the database file

File: server/routes/auth.js [hunk 3]
```diff
@@ -4,5 +4,5 @@ import { validateSignupParams, validateSigninParams } from '../middleware/valida
 
 const router = express.Router();
 router.post('/signup', validateSignupParams, signup);
-router.post('/signin', validateSigninParams, signin);
+// router.post('/signin', validateSigninParams, signin);
```
Comment (mcaleb808) [inline server/routes/auth.js]: @T2Wil Kindly avoid pushing commented code

**Review by mcaleb808 — CHANGES_REQUESTED** [review]

**Comment by mcaleb808** [comment]
@T2Wil Please remember to add build checks(travis and coverage)

File: server/controllers/authController.js (hunk 1 above)
Comment (T2Wil) [reply server/controllers/authController.js]: @mcaleb808 solved!

File: server/controllers/authController.js (hunk 2 above)
Comment (T2Wil) [reply server/controllers/authController.js]: @mcaleb808 Solved!

File: server/routes/auth.js (hunk 3 above)
Comment (T2Wil) [reply server/routes/auth.js]: @mcaleb808 Solved!

**Review by mcaleb808 — APPROVED** [review]

**Pull request merged**